c_glib/arrow-glib/error.cpp

```diff
@@ -37,78 +37,68 @@ G_BEGIN_DECLS
  * values.
  */
 
-G_DEFINE_QUARK(garrow-error-quark, garrow_error)
+G_DEFINE_QUARK(garrow - error - quark, garrow_error)
```

Comment:
This should not be included spaces around `-`.
`G_DEFINE_QUARK` is a macro. `garrow-error-quark` is used as an identifier.

Arrow GLib doesn't use the same coding style as Arrow C++. So we don't need to use `clang-format`.
I'll send a pull request to revert this change.